relayer-cli/src/conclude.rs

```diff
@@ -69,11 +69,7 @@ pub fn exit_with(out: Output) {
     let status = out.status;
 
     // Handle the output message
-    if json() {
-        println!("{}", serde_json::to_string(&out.into_json()).unwrap());
-    } else {
-        println!("{}: {}", out.status, out.result);
-    }
+    println!("{}", String::from(out));
```

Comment:
Just a minor suggestion - I think it would be nice to write success messages to `stdout` (using `println!()`) and errors to `stderr` (using `eprintln!()`).